### PR TITLE
ref: Loading modal slowness

### LINKED ISSUE #8406: Loading modal slowness
if i wait for all my balances to load and then do any action, it loads again, what is it loading for? balances? collection floor? 

https://github.com/kodadot/nft-gallery/assets/36627808/e045c6a3-0d93-43f9-8f90-987dc6eef462

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #8406

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

https://github.com/kodadot/nft-gallery/assets/44554284/fa9eab1d-9005-42ba-a1bd-c9e95220a516

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at d007161</samp>

Renamed `hasBalances` to `isReady` in the auto-teleport feature logic and components to improve clarity and accuracy. Refactored the fee fetching and tracking logic in the `useAutoTeleportTransitionDetails` composable function to enable more consistent fee calculation and transition readiness. Updated the loading state of the modals and buttons that use the auto-teleport feature to reflect the `isReady` property.

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at d007161</samp>

> _`isReady` checks_
> _auto-teleport feature_
> _snowy winter code_

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
sorry, but its not even loading here for me now
showed the console this time, lots of bsx errors

https://github.com/kodadot/nft-gallery/assets/36627808/9e0fe95c-d34f-442c-9040-c8b7115e42a1

**Comment by hassnian** [comment]
> sorry, but its not even loading here for me now showed the console this time, lots of bsx errors
> 
>  loading.mp4

bsx issue @prury pls try again


> wild `isReady` appears

@yangwao  couldn't find that , where did you see that , minting , buying or listing

File: composables/autoTeleport/useAutoTeleportTransitionDetails.ts [hunk 1]
```diff
@@ -202,6 +211,15 @@ export default function (
...
     async () => {
+      console.log(
+        'action' + actionAutoFees.value,
+        fees.actionAutoFees,
+        fees.actionLazyFetch,
+        needsSourceChainBalances.value,
+        feesParams,
+        fees,
+      )
+
```
Comment (roiLeo) [inline composables/autoTeleport/useAutoTeleportTransitionDetails.ts]:
```suggestion
```

**Review by roiLeo — APPROVED** [review]

**Comment by prury** [comment]
way faster now!

**Comment by hassnian** [comment]
> way faster now! only thing i noticed is lots of console errors that weren't occurring on canary(except for `[AUTOTELEPORT]`): image

fixed a bug that caused the AUTOTELEPORT error message to appear for nfts with royalties.

can't reproduce the other errors, can you send a video,

thanks

**Comment by prury** [comment]
> > way faster now! only thing i noticed is lots of console errors that weren't occurring on canary(except for `[AUTOTELEPORT]`): image
> 
> fixed a bug that caused the AUTOTELEPORT error message to appear for nfts with royalties.
> 
> can't reproduce the other errors, can you send a video,
> 
> thanks

no more errors now, ty!

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 7.16 USD/DOT ~ 6.983 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0xf6ca75b6b6001733b04c7a595c17c9643da57fc43ee37bf1e39807188f6462af](https://polkadot.subscan.io/extrinsic/0xf6ca75b6b6001733b04c7a595c17c9643da57fc43ee37bf1e39807188f6462af)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**